src/app/community-list-page/community-list-service.ts

```diff
@@ -99,6 +99,8 @@ const communityListStateSelector = (state: AppState) => state.communityList;
 const expandedNodesSelector = createSelector(communityListStateSelector, (communityList: CommunityListState) => communityList.expandedNodes);
 const loadingNodeSelector = createSelector(communityListStateSelector, (communityList: CommunityListState) => communityList.loadingNode);
 
+export const MAX_COMCOLS_PER_PAGE = 2;
```

Comment:
This is likely left on 2 for testing during development. Since we want to avoid pagination inside the tree to only where it is really necessary for performance, I'd change this to at least 50